File: Fall-2016/HW-3/p6.c
/*
 * In The Name Of God
 * ========================================
 * [] File Name : p6.c
 *
 * [] Creation Date : 08-11-2016
 *
 * [] Created By : Parham Alvani (parham.alvani@gmail.com)
 * =======================================
*/
/*
 * Copyright (c) 2016 Parham Alvani.
*/
#include <stdio.h>

int main(int argc, char *argv[])
{
	int n;
	int a, b, c, d, e;

	scanf("%d", &n);

	if (n < 10000 || n > 99999) {
		printf("Error\n");
		return 0;
	}

	a = n % 10;
	n /= 10;
	b = n % 10;
	n /= 10;
	c = n % 10;
	n /= 10;
	d = n % 10;
	n /= 10;
	e = n % 10;

	if (a == e && b == d)
		printf("True\n");
	else
		printf("False\n");
}

File: Fall-2019/HW-4/p6/p6.go
/*
 *
 * In The Name of God
 *
 * +===============================================
 * | Author:        Parham Alvani <parham.alvani@gmail.com>
 * |
 * | Creation Date: 14-11-2019
 * |
 * | File Name:     p6.go
 * +===============================================
 */

package main

import "fmt"

func main() {
	var n int
	var g int
	var w int

	fmt.Scanf("%d %d\n", &n, &g)
	for i := 0; i < n; i++ {
		var h int
		fmt.Scanf("%d", &h)
		if h > g {
			w += 2
		} else {
			w++
		}
	}

	fmt.Println(w)
}
